TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteListFragment.kt

```diff
@@ -29,7 +33,7 @@ import java.io.ObjectOutputStream
 import java.lang.Exception
 import java.time.LocalDateTime
 
-class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
```

Comment:
If we're renaming this, should we rename "NoteFragment" to "NoteDetailFragment"?

Reply:
good point!